doc/Reference/Reactive/feed.md

```diff
@@ -1,93 +1,43 @@
-# Feed
+# What are feeds?
 
-## Sources: How to create a feed
+Feeds are there to manage asynchronous operations (for example requesting data from a service) and expose the result to the View in an efficient manner.
 
-You can build a _feed_ from different sources. The main entry point is the static class `Feed`:
+They provide out of the box support for task-based methods as well as [Async-Enumerables](https://learn.microsoft.com/dotnet/api/system.collections.generic.iasyncenumerable-1) ones.
 
-Given an `IWeatherService` and `ILocationService`:
+Feeds include additional metadata that indicates whether the operation is still in progress, ended in an error, or if it was successful, whether the data that was returned contains any entries or was empty.
 
-```csharp
-public interface IWeatherService
-{
-    /// <summary>
-    /// Asynchronously gets the current weather
-    /// </summary>
-    Task<WeatherInfo> GetCurrentWeather(CancellationToken ct);
-
-    /// <summary>
-    /// Asynchronously gets the details of a weather alert.
-    /// </summary>
-    Task<string> GetAlertDetails(WeatherAlert alert, CancellationToken ct);
-
-    /// <summary>
-    /// Asynchronously gets the weather forecast for the given day
-    /// </summary>
-    Task<WeatherInfo> GetWeatherForecast(DateTime date, CancellationToken ct);
-}
-
-public record WeatherInfo(double Temperature, WeatherAlert? Alert);
-
-public record WeatherAlert(Guid Id, string? Title);
-
-public interface ILocationService
-{
-    /// <summary>
-    /// Asynchronously gets the name of the current city.
-    /// </summary>
-    Task<string> GetCurrentCity(CancellationToken ct);
-
-    /// <summary>
-    /// Gets teh list of all supported cities.
-    /// </summary>
-    Task<ImmutableList<string>> GetCities(CancellationToken ct);
-}
-```
+## Feeds are stateless
+
+Feeds are typically used to request data from services and expose it in a stateless manner so that the resulting data can be displayed by the View.
+
+Feeds are stateless and do not provide support for reacting to changes the user makes to the data on the View. The data can only be reloaded and refreshed upon request which is when the underlying task or Async-Enumerable will be invoked and the data refreshed. In other words, a feed is a read-only representation of the data received from the server.
+
+In contrast to feeds, [states](xref:Uno.Extensions.Mvux.States) (`IState` or `IListState`), as the name suggests, are stateful and keep track of the latest value, as updates are applied.
+
+## Sources: How to create a feed
 
 ### Async
 
-Creates a feed from an async method.
+Feeds can be created from an async method using the Feed.Async factory method. Here’s an example:
 
 ```csharp
-private IWeatherService _weatherService;
-
-public IFeed<WeatherInfo> Weather => Feed.Async(async ct => await _weatherService.GetCurrentWeather(ct));
+public static IFeed<T> Async<T>(Func<CancellationToken, Task<T>> asyncFunc, Signal? refreshSignal = null);
```

Comment:
I think the code snippet should show how to use the `Async` method. In the previous docs we had:

```csharp
public IFeed<CounterValue> Value => Feed.Async(_myService.CountOne);
```
and
```csharp
private IWeatherService _weatherService;
public IFeed<WeatherInfo> Weather => Feed.Async(async ct => await _weatherService.GetCurrentWeather(ct));
```